core/src/main/java/tc/oc/pgm/api/integration/NickIntegration.java

```diff
@@ -0,0 +1,10 @@
+package tc.oc.pgm.api.integration;
+
+import javax.annotation.Nullable;
+import org.bukkit.entity.Player;
+
+public interface NickIntegration {
+
+  @Nullable
+  String getNick(Player player);
```

Comment:
`UUID` instead of `Player`

Reply:
This is because we should probably pre-fetch the nick on async login, when `Player` does not exist yet.